rholang/examples/arithmetic_test.rho

```diff
@@ -0,0 +1,14 @@
+// -25, false, true, -0.4
+
+new arithmetic_test in {
+    contract arithmetic_test(rtn) = {
+        { (5+(-5))*5/1+6+9-40 } |
```

Comment:
How about shoving these expressions into an array and then matching against the expected resulting array?  (Ditto below)

Reply:
Also file a ticket to add tests and integrate with the continuous build.

Reply:
Actually is the reimplementation effort done to the point in which I can check the outputs of the tests?

Reply:
I've added the ticket, regardless.

Reply:
I think I'll rewrite the arithmetic_test to actually be unit test-like. I'm actually now not sure of some of the behavior - like what do exceptions in Rholang look like for divide by zero or what does overflow look like? Maybe we should have a separate PR for this?

Reply:
I've addressed this portion specifically in https://github.com/rchain/rchain/pull/37

Reply:
> Actually is the reimplementation effort done to the point in which I can check the outputs of the tests?

No, reimplementation isn't to that point.  You should write code to invoke an external VM on the RBL and check the result.  Once the scala VM is to the point where it can run the tests, we just change the filename of the VM in your test framework.

But yeah, if you want to put this off for another PR, that's fine.  Just be sure there's a ticket for it.

Reply:
@pyrocto Yeah unfortunately "You should write code to invoke an external VM on the RBL and check the result" is not trivial if you're talking about the C++ implementation. This has been an issue for a couple months but I can't get the binary for the C++ implementation to run.

Reply:
I've filed the ticket for actually creating a test framework that runs the VM on the compiler output  here https://rchain.atlassian.net/browse/RHOL-22 , and the ticket for updating the comparison_test here https://rchain.atlassian.net/browse/RHOL-28 .